### PR TITLE
Nada/style issues

### PR DESCRIPTION
**Pull request by nada-deriv**
Style issues

Screenshot 2024-06-12 at 11 50 40 AM
Screenshot 2024-06-12 at 11 51 28 AM
Screenshot 2024-06-12 at 11 51 45 AM
Screenshot 2024-06-12 at 11 55 16 AM
Screenshot 2024-06-12 at 11 52 41 AM

https://github.com/deriv-com/p2p/assets/122768621/6650a251-be27-480f-b11c-8c3c7d736f40

### CODE REVIEW

File: src/components/PaymentMethodForm/PaymentMethodForm.scss [hunk 1]
```diff
@@ -23,6 +23,17 @@
...
+            & .payment-method-form__fields {
+                & .payment-method-form__field-wrapper {
+                    & .deriv-input__container {
+                        & .deriv-input__right-content {
+                            display: flex;
+                            align-items: center;
+                        }
+                    }
+                }
+            }
```
Comment (ameerul-deriv) [inline src/components/PaymentMethodForm/PaymentMethodForm.scss:36]: is it possible to just target the .deriv-input__right-content inside of the field wrapper below on line 130

File: src/pages/my-ads/components/AdType/AdType.tsx [hunk 2]
```diff
@@ -1,19 +1,20 @@
...
 type TAdTypeProps = {
     adPauseColor: TTextColors;
     floatRate: string;
 };
 const AdType = ({ adPauseColor, floatRate }: TAdTypeProps) => {
+    const { isDesktop } = useDevice();
     return (
         <div className='ad-type'>
-            <Text as='span' className='ad-type__badge' color={adPauseColor} size='2xs'>
+            <Text as='span' className='ad-type__badge' color={adPauseColor} size={isDesktop ? 'xs' : 'sm'}>
```
Comment (ameerul-deriv) [inline src/pages/my-ads/components/AdType/AdType.tsx:14]: could you create a const variable for the size

File: src/pages/my-ads/components/AdTypeSection/AdTypeSection.scss [hunk 3]
```diff
@@ -1,7 +1,19 @@
...
+        }
+    }
     @include mobile {
         padding: 0 1.6rem;
         overflow: auto;
         max-height: calc(100vh - 27rem);
+
+        &--edit {
+            padding-top: 2.4rem;
+        }
```
Comment (ameerul-deriv) [inline src/pages/my-ads/components/AdTypeSection/AdTypeSection.scss:17]: is this being affected by the mobile padding: 0 1.6rem? if it is could we do this?
```suggestion
        padding: 2.4rem 1.6rem 0;
        overflow: auto;
        max-height: calc(100vh - 27rem);
```

Comment (nada-deriv) [reply src/pages/my-ads/components/AdTypeSection/AdTypeSection.scss:17]: but its needed only for edit.

Comment (nada-deriv) [reply src/pages/my-ads/components/AdTypeSection/AdTypeSection.scss]: i think i need to change the order of the styles.. i'll update it

**Review by ameerul-deriv — APPROVED** [review]

**Review by farrah-deriv — APPROVED** [review]

**Pull request merged**